### ISSUE TITLE
Save to EPUB v3

### ISSUE THREAD

**Issue by w64bit** [benchen71/epub-metadata-editor#84]
Can an option be added to the program to save all (old) files to EPUB v3?
Thank you

**Comment by benchen71** [comment]
No, sorry. That would be a mammoth task that is well beyond a single developer!
